### PR TITLE
Design rank

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/dlgRank.vb [hunk 1]
```diff
@@ -30,74 +30,69 @@ Public Class dlgRank
...
     End Sub
 
     Private Sub InitialiseDialog()
         ucrBase.iHelpTopicID = 25
 
         'Setting Parameters and Data types allowed
+        ucrReceiverRank.SetParameter(New RParameter("x", 0))
         ucrReceiverRank.Selector = ucrSelectorForRank
         ucrReceiverRank.SetMeAsReceiver()
         ucrReceiverRank.SetIncludedDataTypes({"numeric"})
```
Comment (lilyclements) [inline instat/dlgRank.vb]: @rdstern @dannyparsons How come we are only allowing numeric columns to be ranked? I checked on R and it allows any column, and works quite nicely for factors.
Can I remove this line?

Comment (dannyparsons) [inline instat/dlgRank.vb]: yes good spot, this seems to work for any column.

Comment (lilyclements) [inline instat/dlgRank.vb]: Great, removed this line. Thanks

**Comment by lilyclements** [comment]
@dannyparsons this is ready to merge

**Pull request merged**